trip_app/trips/views.py

```diff
@@ -69,21 +80,29 @@ def get_queryset(self):
 
     @action(detail=True, methods=["post"])
     def reserve(self, request, pk=None):
```

Comment:
Hm..I think that I have an idea... Do you mean sort of race conditions that can happen in case of concurrent requests hitting the same resource ? In this case I should think of a lock... I haven't even thought about it before.. Thanks.